src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -99,6 +103,19 @@ def create_subvolume(self, spec, size=None, namespace_isolated=True, mode=0o755,
                 xattr_val = self._get_ancestor_xattr(subvolpath, "ceph.dir.layout.pool")
             # TODO: handle error...
             self.fs.setxattr(subvolpath, xattr_key, xattr_val.encode('utf-8'), 0)
+
+            if uid is not None:
+                try:
+                    self.fs.setxattr(subvolpath, 'ceph.dir.layout.uid', uid.encode('utf-8'), 0)
```

Comment:
Um, not sure where "ceph.dir.layout.uid" comes from. What I was expecting was `self.fs.chown(...)`?

Reply:
I will introduce that in cephfs.pyx and then call it.